### ISSUE TITLE
Layer links do not work

### ISSUE THREAD

**Issue by isaisabel** [mitre-attack/attack-website#169]
Links to view the layer on the NAvigator includes the domain twice. For example:
```
https://mitre-attack.github.io/attack-navigator/beta/enterprise/enterprise/#layerURL=
```
This is because the navigator instances in the config already specifies the domain URL, but the code still appends the domain suffix to the URL in the navigator macro:

https://github.com/mitre-attack/attack-website/blob/develop/attack-theme/templates/macros/navigator.html#L21

Removing the `{{layer.domain}}` suffix should fix the issue.

See also this commit for a manual patch fixing the built content: https://github.com/mitre-attack/attack-website/commit/36d72f6014aaa6f2635f879c6f942571fd4aa8f2

**Comment by isaisabel** [comment]
Reopening so that this fix can be added to master and sub-techniques versions

**Comment by isaisabel** [comment]
fixed in v1.3.1
